core/services/chainlink/config_general.go

```diff
@@ -345,6 +345,11 @@ func (g *generalConfig) StarkNetEnabled() bool {
 	return false
 }
 
+// TODO: Once Aptos config is ready for public repo, update this method
+func (g *generalConfig) AptosEnabled() bool {
+	return false
+}
```

Comment:
No, we want to be able to test it through LOOP.

```suggestion
func (g *generalConfig) AptosEnabled() bool {
	for _, c := range g.c.Aptos {
		v, ok := c["Enabled"]
		if !ok {
			return false
		}
		return v == true
	}
	return false
}
```

Something like this, but needs testing since `v` could be typed as `*bool`

Reply:
Resolved this, thanks for the idea @archseer